docs/getting-started/running-tests.md

```diff
@@ -9,14 +9,15 @@ components hooked up, and your test cases written.
 
 ## Importing your specs
 
-The final step before you can run your tests is to import them in `index.test.js`
-and pass them to the Cavy Tester.
+The final step before you can run your tests is to import them and pass them to
+the Cavy Tester.
 
-Open `index.test.js`, import your tests, and replace the `ExampleSpec` in the 
-Tester's `specs` prop with your specs:
+Open `index.test.js` (or `index.{ios,android}.js` for non-cli users), import
```

Comment:
I guess this is a bit presumptive - they could call their entry point lots of things!

Reply:
Agreed